WordPress/src/main/java/org/wordpress/android/ui/me/MyProfileActivity.java

```diff
@@ -0,0 +1,143 @@
+package org.wordpress.android.ui.me;
+
+import android.os.Bundle;
+import android.support.v7.app.ActionBar;
+import android.support.v7.app.AppCompatActivity;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.TextView;
+
+import org.wordpress.android.R;
+import org.wordpress.android.models.Account;
+import org.wordpress.android.models.AccountHelper;
+import org.wordpress.android.ui.ActivityLauncher;
+import org.wordpress.android.util.DialogUtils;
+import org.wordpress.android.util.StringUtils;
+import org.wordpress.android.widgets.WPTextView;
+import org.xmlrpc.android.ApiHelper;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class MyProfileActivity extends AppCompatActivity {
+
+    private WPTextView mFirstName;
+    private WPTextView mLastName;
+    private WPTextView mDisplayName;
+    private WPTextView mAboutMe;
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        AccountHelper.getDefaultAccount().fetchAccountSettings(new ApiHelper.GenericCallback() {
+            @Override
+            public void onSuccess() {
+                refreshDetails();
+            }
+
+            @Override
+            public void onFailure(ApiHelper.ErrorType errorType, String errorMessage, Throwable throwable) {
+            }
+        });
+
+        ActionBar actionBar = getSupportActionBar();
+        if (actionBar != null) {
+            actionBar.setHomeButtonEnabled(true);
+            actionBar.setDisplayHomeAsUpEnabled(true);
+        }
+        setContentView(R.layout.my_profile_activity);
+
+        setTitle(R.string.my_profile);
```

Comment:
It's fine to set the title here, but note that you can set the title in the manifest entry for this activity instead.